### PR TITLE
added email specific documentation for actions and events. also added…

### PR DESCRIPTION
**Pull request by CrystalOnScript**
… missing actions.

Per [AMP4Email: Standardize email meta whitelists](https://github.com/ampproject/amphtml/issues/27094)

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Is there a way we could avoiding duplicating the action doc?

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+
+<!--
+This file is imported from https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md.
+Please do not change this file.
+If you have found a bug or an issue please
+have a look and request a pull request there.
+-->
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (caroqliu) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md:20]: 2020 :)

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
+---
+$title: Actions and events
+order: 7
```
Comment (caroqliu) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md:3]: For my own learning, what does `order: 7` mean?

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+
+
+
```
Comment (caroqliu) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md:38]: Are these many extra lines necessary?

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+    <td>Toggles the visibility of the target element. If an
+    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus"><code>autofocus</code> element</a> becomes visible as a
+    result, it gains focus.</td>
+  </tr>
+  <tr>
+    <td><code>toggleClass(class=STRING, force=BOOLEAN)</code></td>
+    <td>Toggles class of the target element. <code>force</code> is optional, and if defined, it ensures that class would only be added but not removed if set to <code>true</code>, and only removed but not added if set to <code>false</code>.</td>
+  </tr>
+  <tr>
+    <td><code>scrollTo(duration=INTEGER, position=STRING)</code></td>
```
Comment (caroqliu) [inline pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md:363]: Please remove `scrollTo` section

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md (hunk 3 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md:38]: I've removed them, we'll see if prettifier puts them back!

File: pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md (hunk 2 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md]: Sure! This is the order the document appears in the sidebar under "Learn" - which appears to be inaccurate...

**Comment by CrystalOnScript** [comment]
Thanks for the great review @caroqliu ! It flagged that this document is actually being imported from the amphtml repo (whoops!). But there also appears to be some issues with the the order the importer placed them in TOC. I'll try to flag this with the team. 

This PR is now dependent on [amphtml #28144](https://github.com/ampproject/amphtml/pull/28144).

**Comment by CrystalOnScript** [comment]
ping @sebastianbenz

**Pull request merged**